agent/src/main/scala/za/co/absa/atum/agent/model/Measure.scala

```diff
@@ -50,107 +50,107 @@ object Measure {
       ResultOfMeasurement(resultValue, resultValueType)
     }
   }
   object SumOfValuesOfColumn extends MeasureType {
-    def apply(controlCol: String): SumOfValuesOfColumn = {
-      SumOfValuesOfColumn(controlCol, measureName, resultValueType)
+    def apply(measuredCol: String): SumOfValuesOfColumn = {
+      SumOfValuesOfColumn(measuredCol, measureName, resultValueType)
     }
 
     override val measureName: String = "aggregatedTotal"
     override val resultValueType: ResultValueType.ResultValueType = ResultValueType.BigDecimal
   }
 
   case class AbsSumOfValuesOfColumn private (
-    controlCol: String,
-    measureName: String,
-    resultValueType: ResultValueType.ResultValueType
+                                              measuredColumn: String,
+                                              measureName: String,
+                                              resultValueType: ResultValueType.ResultValueType
   ) extends Measure {
 
     override def function: MeasurementFunction = (ds: DataFrame) => {
       val aggCol = sum(abs(col(valueColumnName)))
-      val resultValue = aggregateColumn(ds, controlCol, aggCol)
+      val resultValue = aggregateColumn(ds, measuredColumn, aggCol)
       ResultOfMeasurement(resultValue, resultValueType)
     }
   }
   object AbsSumOfValuesOfColumn extends MeasureType {
-    def apply(controlCol: String): AbsSumOfValuesOfColumn = {
-      AbsSumOfValuesOfColumn(controlCol, measureName, resultValueType)
+    def apply(measuredCol: String): AbsSumOfValuesOfColumn = {
+      AbsSumOfValuesOfColumn(measuredCol, measureName, resultValueType)
     }
 
     override val measureName: String = "absAggregatedTotal"
     override val resultValueType: ResultValueType.ResultValueType = ResultValueType.Double
   }
 
   case class SumOfHashesOfColumn private (
-    controlCol: String,
-    measureName: String,
-    resultValueType: ResultValueType.ResultValueType
+                                           measuredColumn: String,
+                                           measureName: String,
+                                           resultValueType: ResultValueType.ResultValueType
   ) extends Measure {
 
     override def function: MeasurementFunction = (ds: DataFrame) => {
 
       val aggregatedColumnName = ds.schema.getClosestUniqueName("sum_of_hashes")
       val value = ds
-        .withColumn(aggregatedColumnName, crc32(col(controlCol).cast("String")))
+        .withColumn(aggregatedColumnName, crc32(col(measuredColumn).cast("String")))
         .agg(sum(col(aggregatedColumnName)))
         .collect()(0)(0)
       val resultValue = if (value == null) "" else value.toString
       ResultOfMeasurement(resultValue, ResultValueType.String)
     }
   }
   object SumOfHashesOfColumn extends MeasureType {
-    def apply(controlCol: String): SumOfHashesOfColumn = {
-      SumOfHashesOfColumn(controlCol, measureName, resultValueType)
+    def apply(measuredCol: String): SumOfHashesOfColumn = {
```

Comment:
Why not to rename to `measuredColumn` to keep it consistent all around?

Reply:
true point, done